packages/shared/src/components/feeds/FeedNav.tsx

```diff
@@ -123,6 +127,29 @@ function FeedNav(): ReactElement {
     isCustomDefaultFeed,
   ]);
 
+  const previousScrollY = React.useRef(0);
+
+  useEventListener(globalThis, 'scroll', () => {
+    // when scrolled down we should hide the header
+    // when scrolled up, we should bring it back
+    const { scrollY } = window;
+    const shouldHeaderBeVisible = scrollY < previousScrollY.current;
+
+    previousScrollY.current = scrollY;
+
+    if (shouldHeaderBeVisible === isHeaderVisible) {
+      return;
+    }
+
+    if (!shouldHeaderBeVisible && scrollY < 60) {
```

Comment:
This `60` feels like something we can forget while changing header height, can we somehow extract it in const, or use existing const?

Reply:
Yeah, putting a clear name would make it easier to understand.